internal/providers/terraform/azure/app_service_environment.go

```diff
@@ -0,0 +1,101 @@
+package azure
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+
+	"github.com/shopspring/decimal"
+)
+
+func GetAzureRMAppIsolatedServicePlanRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_app_service_environment",
+		RFunc: NewAzureRMAppIsolatedServicePlan,
+		ReferenceAttributes: []string{
+			"resource_group_name",
+		},
+		Notes: []string{
+			"Costs associated with running an app service plan in Azure",
+		},
```

Comment:
```suggestion
```

This isn't needed since the notes are only to document cases that could not be estimated.